mantle/cmd/kola/testiso.go

```diff
@@ -388,3 +402,27 @@ func testLiveIso(inst platform.Install, completionfile string) error {
 
 	return awaitCompletion(mach.QemuInst, completionChannel, []string{liveOKSignal, signalCompleteString})
 }
+
+func testLiveLogin() error {
+	builddir := kola.CosaBuild.Dir
+	isopath := filepath.Join(builddir, kola.CosaBuild.Meta.BuildArtifacts.LiveIso.Path)
+	builder := newBaseQemuBuilder()
+	// See AddInstallISO, but drop the bootindex bit; we want it to be the default
+	builder.Append("-drive", "file="+isopath+",format=raw,if=none,readonly=on,id=installiso", "-device", "ide-cd,drive=installiso")
+
+	completionChannel, err := builder.VirtioChannelRead("coreos.liveiso-success")
+	if err != nil {
+		return err
+	}
+
+	// No network device to test https://github.com/coreos/fedora-coreos-config/pull/326
+	builder.Append("-net", "none")
+
+	mach, err := builder.Exec()
+	if err != nil {
+		return errors.Wrapf(err, "running iso")
+	}
+	defer mach.Destroy()
+
+	return awaitCompletion(mach, completionChannel, []string{"coreos-liveiso-success"})
```

Comment:
This jumped out at me since the rest had `mach.QemuInst` ... but since this is running instead of installing I assume it's meant to be different.

Reply:
Yep, it's not an install.